pkg/tests/api/alerting/api_backtesting_test.go

```diff
@@ -53,8 +53,7 @@ func TestBacktesting(t *testing.T) {
 	require.Truef(t, ok, "The data file does not contain a field `query`")
 
 	for _, query := range queryRequest.Data {
-		isExpr, _ := query.IsExpression()
-		if isExpr {
+		if query.DatasourceUID == "__expr__" {
```

Comment:
This is the test file. API model doesn't have method IsExpression. Therefore, I used this condition because I create the input data as well. The method IsExpression of the domain AlertQuery should be used anywhere else.